### PR TITLE
Fix TreeIter breakage

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

**Comment by gkoz** [comment]
This needs to be merged for https://github.com/gtk-rs/gtk/pull/176 to come out green.

File: src/treeview.rs [hunk 1]
```diff
@@ -88,16 +85,14 @@ fn main() {
...
-        right_store.append(&mut iter, None);
+    for i in 0..10 {
+        let iter = right_store.append(None);
         right_store.set_value(&iter, 0, &value);
 
-        let mut child_iter = gtk::TreeIter::new();
-
-        right_store.append(&mut child_iter, Some(&iter));
-        right_store.set_string(&child_iter, 0, "I'm a child node");
+        for _ in (0..i) {
```
Comment (GuillaumeGomez) [inline src/treeview.rs]:
```Rust
for _ in 0..i {
```

instead.

**Comment by GuillaumeGomez** [comment]
Travis doesn't like this one. :/

File: src/treeview.rs (hunk 1 above)
Comment (gkoz) [inline src/treeview.rs]: Hm, right

**Comment by gkoz** [comment]
The two PRs depend on each other so one of them has to be merged first anyway.

**Comment by GuillaumeGomez** [comment]
Ok, then let's start with this one.

**Pull request merged**